dist/ng-formio-helper.js

```diff
@@ -651,6 +662,16 @@ angular.module('ngFormioHelper', ['formio', 'ngFormioGrid', 'ui.router'])
                     $state) {
             return {
               init: function () {
+                init = true;
+                // Format the roles and access for easy usage.
+                (new Formio(appUrl + '/form')).loadForms({params:{limit: 9999999}}).then(function (forms) {
```

Comment:
Just realized that since /form is not publicly available this will fail when not logged in to a user with sufficient permissions. We need to have a conversation about opening this endpoint up more.